docs/dev/new_check_howto.md

```diff
@@ -25,59 +25,62 @@ You'll also need `docker-compose` in order to run the test harness.
 
 ## Setup
 
-Clone the [integrations-extras repository][7] and point your shell at the base directory:
+Clone both the [integrations-extras repository][7] *and* the [integrations-core repository][8]. The Extras repository is where you'll be working, but you'll need tooling from the Core repository to get yourself up and running. By default, that tooling expects you to be working in the `$HOME/dd/` directory — this is optional and can be adjusted via configuration later.
 
+```shell
+mkdir $HOME/dd && cd $HOME/dd       # optional
+git clone https://github.com/DataDog/integrations-extras.git
+git clone https://github.com/DataDog/integrations-core.git
 ```
-git clone https://github.com/DataDog/integrations-extras.git && cd integrations-extras
+
+### Developer toolkit
+
+The [developer toolkit][17] is comprehensive and includes a lot of functionality. Here's what you need to get started:
+
+```
+cd integrations-core
+pip install "datadog-checks-dev[cli]"
 ```
 
-Install the Python packages needed to work on Agent Integrations:
+If you chose to clone these repositories to somewhere other than `$HOME/dd/`, you'll need to adjust the configuration file. The location of the file is dependent on your platform:
 
 ```
-pip install -r requirements-dev.txt
+ddev config find
 ```
 
-[cookiecutter][1] is used to create the skeleton for a new integration:
+Alter the `core` and `extras` options to point to the appropriate directories (don't worry about the other stuff for now).
```

Comment:
Users can actually just do:

`ddev config set extras /path/to/integrations-extras`